Sources/MapboxMaps/Annotations/AnnotationOrchestrator.swift

```diff
@@ -39,16 +39,19 @@ public protocol AnnotationInteractionDelegate: AnyObject {
 
 public class AnnotationOrchestrator {
 
-    private weak var view: UIView?
+    private weak var singleTapGestureRecognizer: UIGestureRecognizer?
```

Comment:
I don't think this needs to be `weak` anymore. `UIGestureRecognizer` only keeps a weak ref to its view, so there's no risk of a cycle.